pyproject.toml

```diff
@@ -15,10 +15,15 @@ dependencies = [
     "signwriting @ git+https://github.com/sign-language-processing/signwriting",
     "signwriting-evaluation @ git+https://github.com/sign-language-processing/signwriting-evaluation.git",
     "google-api-python-client",
-    "pympi-ling"
+    "pympi-ling",
+    "sentencepiece",
+    "pandas"
 ]
 
 [project.optional-dependencies]
+pose_to_signwriting = ["joeynmt @ git+https://github.com/may-/joeys2t.git@4532e8b9b990c8a3c4ccfe9fbd141bddc6d8637b"]
```

Comment:
pandas and sentencepiece are only used by `pose_to_signwriting`

`google-api-python-client` is a `dev` dependency